src/core/arm/dynarmic/arm_dynarmic_64.cpp

```diff
@@ -86,6 +86,27 @@ class DynarmicCallbacks64 : public Dynarmic::A64::UserCallbacks {
                   num_instructions, MemoryReadCode(pc));
     }
 
+    void InstructionCacheOperationRaised(Dynarmic::A64::InstructionCacheOperation op,
+                                         VAddr value) override {
+        switch (op) {
+        case Dynarmic::A64::InstructionCacheOperation::InvalidateByVAToPoU: {
+            constexpr u64 ICACHE_LINE_SIZE = 64;
+
+            u64 cache_line_start = value & ~(ICACHE_LINE_SIZE - 1);
```

Comment:
```suggestion
            const u64 cache_line_start = value & ~(ICACHE_LINE_SIZE - 1);
```

Since we try to make mutability explicit